kbfsdokan/main.go

```diff
@@ -25,6 +25,7 @@ var mountType = flag.String("mount-type", defaultMountType, "mount type: default
 var version = flag.Bool("version", false, "Print version")
 var mountFlags = flag.Int64("mount-flags", int64(libdokan.DefaultMountFlags), "Dokan mount flags")
 var dokandll = flag.String("dokan-dll", "", "Absolute path of dokan dll to load")
+var servicemount = flag.Bool("service-mount", false, "get mount path from serfice")
```

Comment:
Also, flags shared among the different binaries should go in `AddFlags` in `libkbfs/init.go`.